rotkehlchen/db/filtering.py

```diff
@@ -903,3 +903,61 @@ def make(
         filters.append(filter_query.timestamp_filter)
         filter_query.filters = filters
         return filter_query
+
+
+class CustomAssetsFilterQuery(DBFilterQuery):
+    @classmethod
+    def make(
+            cls,
+            and_op: bool = True,
+            order_by_rules: Optional[List[Tuple[str, bool]]] = None,
+            limit: Optional[int] = None,
+            offset: Optional[int] = None,
+            identifier: Optional[str] = None,
+            name: Optional[str] = None,
+            custom_asset_type: Optional[str] = None,
+            started: Optional[Timestamp] = None,
+            swapped_for: Optional[Asset] = None,
+    ) -> 'CustomAssetsFilterQuery':
+        if order_by_rules is None:
```

Comment:
I would force passing `order_by_rules` and make sure that they are not none and the api level. Just a suggestion thought, not sure that it is much better